### DISCUSSION TITLE
Discussion: Is Monika a dog or cat person?

### DISCUSSION THREAD

**Discussion by ghost** [Monika-After-Story/MonikaModDev#8327] [general]
I was thinking about making a PR for #7339, but I figured it would be better to ask for somebody else's opinion. 

If Monika is going to tell the player her preference between dogs and cats aswell, what would it be?

A lot of people would associate cats with introverts and dogs with extroverts. But that's just a stereotype.

As an introvert myself, I can say I prefer dogs. It's just that I feel more connected with them and feel kinda awkward around cats. 

But Monika... It's hard for me to tell, to be honest. Since she's more like an outsider, I thought that maybe she would prefer dogs, because they're more likely to enjoy being outside, going to the park, and some people even jog with their dogs. And since Monika likes to exercise, I thought that she would enjoy a fluffy company in her walks if she was able to come to our reality. And they're usually more extroverted than cats, making you feel more comfortable, with their bright personality.

On the other hand, Monika also likes being inside, sometimes just enjoying a cup of coffee and reading a good book. And cats are the perfect company for that. And, since they're usually quieter than dogs, they would be a calming company for her. And just their presence makes you feel cozy.

And, finally, she could also say that she doesn't have a preference, pointing out each one of their different qualities.

I just want to clarify that having a preference doesn't mean that you hate the other. You could just feel more comfortable with one of them, or both. Preferring dogs doesn't mean that you hate cats, or vice-versa. I often see a lot of people arguing on who's better, but they both deserve love. And, independent on the player's preference or Monika's, I feel like she would say that aswell.

**Comment by KansaiGal** [comment]
Given her reason for being vegetarian... maybe she'd want to choose based on ecological impact?
[Cats can only ever be carnivorous...](https://www.bbc.com/future/article/20200304-can-you-feed-cats-and-dogs-a-vegan-diet)

**Comment by Pyromanic8** [comment]
I remember her mentioning liking dogs in one of her topics before. I don't remember seeing a cat one, but it could very well exist.

**Comment by Booplicate** [comment]
She talks about both
https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/script-topics.rpy#L7214-L7227
https://github.com/Monika-After-Story/MonikaModDev/blob/master/Monika%20After%20Story/game/script-topics.rpy#L7197-L7209

**Comment by ghost** [comment]
@KansaiGal I agree that maybe she would consider that in case she wanted to keep one of them, but what I'm saying it's based on her personal preference and personality. After all, you can have a preference without necessarily having a dog/cat.

**Comment by KansaiGal** [comment]
For sure; but if you're looking for ideas I thought I'd throw that out there.  As for dog vs cat... I don't know.

**Comment by AutumnLeaves991** [comment]
I think that given monika's love for the player, she'd take a neutral stance and after asking the player what their own preference is, would say she wouldnt mind having either as a pet, as long as its with you or something.  After all, she cant really have a pet anyhow unless she makes it to our reality.  

I like the environmentalism point though, so I do think its likely she'd prefer an indoor cat over an outdoor one if the player says they like cats better.  Perhaps she could have a little dialogue about how outdoor cats damage ecosystems or something.. Idk, just spitballing

**Comment by fromulus** [comment]
Even as someone who would pick dogs myself, I feel like the topic of pets is way too vast for us to just keep it at dogs and cats and not have it feel rather close-minded.
What about people who prefer birds or reptiles, what about people who keep insects or spiders as pets ?
And what about people who live with more exotic animals like wolves and such ?

there isn't really a way to represent those people due to how many pet options there are out there.
